.gitmodules

```diff
@@ -74,3 +74,7 @@
 	path = third_party/efr32_sdk/repo
 	url = https://github.com/SiliconLabs/sdk_support.git
 	branch = master
+[submodule "simw-top-mini"]
```

Comment:
the submodule name seems very different than the Github URL. why is that? should we name it 'plug_and_trust' or similar instead?

Reply:
Plug & Trust Middleware is a software package supporting several different microcontrollers on different platforms. The full package is distributed via nxp.com we provide a reduced package (simw-top-mini) under an open-source license to facilitate integration with the CHIP build system.
Preferably, We would like to retain current naming to be consistent between the different packages and its documentation.